src/services/getUser.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+export const  getUser = async (email,password) => {
+  try {
+    const {data,status} = await axios.post("/api/auth/login",{email,password});
+    return {data,status};
+  } catch (error) {
+    console.error("LOGGING IN FAILED",error);
+  }
```

Comment:
In case of `error`, you can return it as well

Reply:
done